src/content/docs/configuration/sharing.mdx

```diff
@@ -93,4 +93,9 @@ pull_request_rules:
 
   - Values in the `default` key will be merged and remote default values will
     apply to local configuration.
+
+  - If there are `pull_request_rules`, `queue_rules`, `priority_rules`, `partition_rules`
+    or `merge_protections` rules with the same name in both the extended config and the
+    config from which it is extended, then the rule, with the same name, in the config that
+    extends the other will NOT be overwritten.
```

Comment:
```suggestion
  - If there are `pull_request_rules`, `queue_rules`, `priority_rules`, `partition_rules`
    or `merge_protections` rules with the same name in both the extended config and the
    config from which it is extended, then the duplicated rule will NOT be overwritten. Thus, the rule
    defined in the config containing the `extends` keyword will be used.
```
I had some troubles understanding, may be we can turn it in another way ?

My suggestion is just an example we can change it to be clearer, wdyt ?